contracts/0.8.9/StakingRouter.sol

```diff
@@ -535,6 +535,13 @@ contract StakingRouter is AccessControlEnumerable, BeaconChainDepositor, Version
         return STAKING_MODULES_COUNT_POSITION.getStorageUint256();
     }
 
+    /**
+     * @dev Returns true if staking module with the given id was registered via `addStakingModule`, false otherwise
+     */
+    function hasStakingModule(uint256 _stakingModuleId) external view returns (bool) {
+        return _stakingModuleId != 0 && _stakingModuleId <= LAST_STAKING_MODULE_ID_POSITION.getStorageUint256();
```

Comment:
Minor: might be replaced with `_getStorageStakingIndicesMapping()[_stakingModuleId] != 0`. Both checks will work equally until we implement the staking module removal feature.